### PR TITLE
Update retry doc

### PR DESCRIPTION
**Pull request by xinlian12**
For multi-master region, SDK will send the session token for write operation as well, so SDK could get 404/1002 as well. The current doc mentioned it will only happen for read operation which is not correct.

### CODE REVIEW

**Review by kushagraThapar — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
...>If the second replica also throws a 404: </br><ul><li>If there are additional regions for the Cosmos DB account: </br><ul><li>For a single-write region account, the client retries the request against the write region for the account (if the first request targeted the read region). The client will follow the same path of targeting 1 replica followed by a retry against another replica if the read against the first replica resulted in a 404. </li><li>If all regions have 2 replicas that returned a 404, the exception is bubbled back up to the calling code. </li></ul></li><ul></li></ul></li></ul></li><li>When using Bounded Staleness and Strong Consistency: </br><ul><li>The SDK sends the read request to 2 replicas for the partition in the specified region. </li><li>If one of the 2 replicas returns a 404, data from the second replica is returned by the SDK. </li><li>If both replicas return a 404, the exception is bubbled up to the application. </li><li>The Cosmos DB SDK does NOT retry the read against a remote region is both replicas in the specified region return a 404 for the point read operation. </li></ul></li></ul></li></ul>                                                           |
```
Comment (kirankumarkolli) [inline sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md:8]: > For a single-write region, N/A as this exception is only encountered for point read operations

How about "... is only applicable for read operations"

File: sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (kirankumarkolli) [inline sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md:8]: 5secons might be interpreted as contract. Is that guaranteed?

File: sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md:8]: will update in next iteration

File: sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md (hunk 2 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md:8]: maybe I can change into: will first retry on the current region, then other regions?

**Review by FabianMeiswinkel — APPROVED** [review]

File: sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md (hunk 1 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md:8]: updated

File: sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md (hunk 2 above)
Comment (xinlian12) [reply sdk/cosmos/azure-cosmos/docs/ErrorCodesAndRetries.md]: removed the 5s reference. Changed to: will first retry on the current region few times, then other regions.

**Pull request merged**